source/android/adaptivecards/src/main/java/com/microsoft/adaptivecards/renderer/layout/HorizontalFlowLayout.java

```diff
@@ -50,9 +47,9 @@ public HorizontalFlowLayout(Context context, AttributeSet attrs, int defStyle) {
 //     *
 //     * TODO: uncomment this constructor once the minimum API level of this project is changed to 21 (Lollipop).
 //     */
-    public HorizontalFlowLayout(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
-        super(context, attrs, defStyleAttr, defStyleRes);
-    }
+//    public HorizontalFlowLayout(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
```

Comment:
testing done?

Reply:
Since we will be going to min-version 19, this will remain commented out.